<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>560</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK KR</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Image processing library</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans CJK TC Light</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundImageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>351</width>
     <height>651</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>541</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>AppImages/Back_g.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>AppImages/Widget.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>backgroundImageLabel</zorder>
  <zorder>label</zorder>
  <zorder>LoginInput</zorder>
  <zorder>label_2</zorder>
  <zorder>LoginButton</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
